### PR TITLE
Feature/guzzle configuration

### LINKED ISSUE #65: SSL verify = false
@christeredvartsen We want to use this extension within local development and we are facing:
```
cURL error 60: SSL certificate problem: self signed certificate (see http://curl.haxx.se/libcurl/c/libcurl-errors.html) (GuzzleHttp\Exception\RequestException)
```

Any workaround for this?

### PR DESCRIPTION
**Pull request by vitalyiegorov**
Closes #65

### CODE REVIEW

**Comment by vitalyiegorov** [comment]
Created feature branch, implemented review changes as mentioned by @christeredvartsen in #66

File: tests/ServiceContainer/BehatApiExtensionTest.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ public function testCanBuildConfiguration() {
 
         $this->assertSame([
             'apiClient' => [
-                'base_uri' => 'http://localhost:8080',
```
Comment (christeredvartsen) [inline tests/ServiceContainer/BehatApiExtensionTest.php]: Keep the trailing comma. Goes for the other two changes in this file as well.

**Comment by christeredvartsen** [comment]
Could you also rebase on `develop`?

**Comment by christeredvartsen** [comment]
The `develop` branch in your fork does not seem to be updated. Could you rebase this branch against `imbo:develop` and push that so the PR is completely up to date?

**Comment by christeredvartsen** [comment]
It seems your fork is in a somewhat weird state btw. Did you already merge this feature-branch into your develop-branch? Anyways, I'll squash the commits and merge them into develop now.

**Pull request merged**